shared/mariadb_instance/main.tf

```diff
@@ -129,6 +129,33 @@ resource "aws_db_instance" "database" {
   }
 }
 
+# Configure a MySQL provider for this instance.
+provider "mysql" {
+  version  = "~> 1.5"
+  endpoint = "${aws_db_instance.database.endpoint}"
+  username = "${aws_db_instance.database.username}"
+  password = "${aws_db_instance.database.password}"
+}
+
+resource "mysql_user" "readonly" {
+  user = "readonly"
+}
+
+resource "mysql_user_password" "readonly" {
+  user = "${mysql_user.readonly.user}"
+
+  # This resource automatically generates a password for the MySQL
```

Comment:
It won't at first, since those are stored under different user names (`longshot_readonly` and `rogue_readonly` respectively). Once I've confirmed these new users are working, I'll update  Lastpass and notify folks to update their credentials before removing them.